### DISCUSSION TITLE
workshop mentioned in cdk live episode

### DISCUSSION THREAD

**Discussion by Analect** [awslabs/generative-ai-cdk-constructs#337] [Q&A] [answered]
Hi Team. Thanks for your work on this great resource. I had been watching this [Building generative AI applications and architectures leveraging AWS CDK Constructs!](https://www.youtube.com/watch?v=NI1F4Xxqyr8), which offers great context to this repo and the [samples](https://github.com/aws-samples/generative-ai-cdk-constructs-samples) one. At the end, it was mentioned that there was a workshop covering topics in here. I had a looked in LinkedIn, as you suggested, but wasn't able to find any mention. Has this been released and, if so, could you point me towards it. Tks.

**Comment by scottschreckengaust** [comment, marked as answer]
We are working on the public release of this workshop, it will be coming soon!

**Reply by Analect** [reply]
thanks!
